### PR TITLE
Add refresh button to logbook

### PR DESCRIPTION
**Pull request by cgarwood**
Adds a refresh button to the Logbook panel to refresh events from the current day. Refresh button is hidden on days in the past as the logbook data shouldn't change for those days.

Fixes https://github.com/home-assistant/home-assistant-polymer/issues/284

### CODE REVIEW

**Comment by c727** [comment]
...or better you calculate today in `_isRefreshHidden` else you could get problems if the day changes or with timezones

File: panels/logbook/ha-panel-logbook.html [hunk 1]
```diff
@@ -73,6 +79,7 @@
           disabled='[[isLoading]]'
           required
         ></vaadin-date-picker>
+        <paper-icon-button id='refresh-logbook' icon='mdi:refresh' on-click='refreshLogbook' hidden$='[[_isRefreshHidden(isLoading, _currentDate)]]'></paper-icon-button>
```
Comment (balloob) [inline panels/logbook/ha-panel-logbook.html]: please add each attribute on a new line.

Comment (balloob) [inline panels/logbook/ha-panel-logbook.html]: ID seems not to be used.

Comment (balloob) [reply panels/logbook/ha-panel-logbook.html]: Yes, classes are always preferred. Because IDs have to be unique, you will have to change the HTML if you want to reuse the style for another element.

**Comment by balloob** [comment]
Can you add screenshots.

**Comment by cgarwood** [comment]
ha-logbookrefresh

**Comment by cgarwood** [comment]
Like this?
ha-logbookrefresh2

**Comment by c727** [comment]
here are some improvements for the current status: https://hastebin.com/hilepajaba.xml

Now I think we should always show the refresh button, else on 00:01 we can't refresh yesterdays data

File: panels/logbook/ha-panel-logbook.html [hunk 2]
```diff
@@ -106,10 +112,11 @@
       // ISO8601 formatted date string
       _currentDate: {
         type: String,
-        value: function () {
-          var value = new Date();
-          var today = new Date(Date.UTC(value.getFullYear(), value.getMonth(), value.getDate()));
-          return today.toISOString().split('T')[0];
+        value: () => {
```
Comment (balloob) [inline panels/logbook/ha-panel-logbook.html]: What's this change doing? It does not seem related to the refresh button nor is it a simple conversion of var => const

Comment (cgarwood) [reply panels/logbook/ha-panel-logbook.html]: TBH, I'm not completely sure, it was requested by @c727 above. Something to do with timezones apparently?

Comment (c727) [reply panels/logbook/ha-panel-logbook.html]: toISOString() does not return the local date

Comment (balloob) [reply panels/logbook/ha-panel-logbook.html]: as far as I know, the backend parses the date as a UTC date. So that seems right?

Comment (c727) [reply panels/logbook/ha-panel-logbook.html]: `_cuttentDate` is the date selected in the picker.
`_computeFilterDate(_cuttentDate)` is the ISO formatted string to request the data

Comment (c727) [reply panels/logbook/ha-panel-logbook.html]: hmz nevermind... using the `Date.UTC()` aslo produces correct dates

Sorry^^

**Review by balloob — APPROVED** [review]
